### DISCUSSION TITLE
11/28 생각 나누기

### DISCUSSION THREAD

**Discussion by Orm-camp** [Orm-camp/blog-feedback#40] [동글동글 3기 피드백]
오늘 함께 읽은 글에 대해 드는 생각/의견 자유롭게 나눠주세요 💟 
(공감 가는 의견에는 이모지를, 궁금한 부분이 있다면 대댓글도 남겨보세요!)

**Comment by mh0223** [comment]
요약 잘하는 사람의 특징 중 5번째, "의도를 먼저 파악하여 맥락을 잘 짚는다." 라는 말이 중요하다는 생각이 듭니다. 이 부분에 있어서 어렵게 느껴질 수도 있는 것 같아요. 또한 요약 능력 향상을 위한 팁인, '키워드/핵심문장/주제문 3가지 찾기' 습관을 적용해봐야겠다는 생각이 들었습니다.

**Comment by hansol-woff** [comment]
말할 때나 글 쓸 때나 구구절절하는 편이라서 해당 글을 골랐는데, 딱 정리해서 전달하는건 항상 어려운 것 같습니다 🥲

**Comment by structified** [comment]
매번 필요 이상으로 무겁게 여행 가방을 챙기고, 후회하는 제 모습이 생각났습니다..

**Comment by oratio100** [comment]
글을 읽는 입장에서도 쓰는 입장에서도 맥락을 잘 짚는게 중요한 것 같습니다. 글을 쓰다보면 간혹 맥락을 잃어버릴때가 있습니다. 정보를 욱여넣다가 처음에 글을 쓰기 시작한 작성 의도와는 다른 글이 될 때가 있는데 이 글에서 나온 4가지 요소를 잘 기억해두어야겠습니다.

**Comment by kimjigyun** [comment]
나는 평소 삭제,압축,정리를 잘 못하는거 같은데,, 고수가 되고싶다.

평소에도 키워드/핵심문장/주제문 으로 3개 찾는 습관 적용할수있으면 너무 좋을것 같다.

**Comment by Devkdy22** [comment]
맥락을 짚는 것, 불필요한 내용을 없앨 줄 아는 것, 중점적인 내용을 뽑아낼 수 있는 것 등,, 요약을 잘하는 특징들을 다시 한 번 깨우치며, 글을 쓸땨나 코드를 작성할 때에도 해당 특징들을 생각하며 잘 활용해봐야겠다는 생각이 들었습니다.

**Comment by naseulgee** [comment]
마지막에 첫째~넷째 로 한줄 요약해주신게 역시 요약을 잘하시는 분... 이라는 생각이 들었습니다
앞으로 저도 글을 쓸 때 참고해봐야 겠습니다

**Comment by mycreature** [comment]
글을 쓸때 참고하기 좋은 내용 같습니다. 다만 적응할 시간이 필요할거같습니다
